nvm.sh

```diff
@@ -2997,7 +3034,7 @@ nvm() {
 
       while [ $# -ne 0 ]; do
         case "$1" in
-          --silent) NVM_USE_SILENT=1 ;;
+          "--silent") NVM_USE_SILENT=1 ;;
```

Comment:
```suggestion
          --silent) NVM_USE_SILENT=1 ;;
```
the quotes are unnecessary.